Weather App/WeathersList/view/DayForecastWeatherCell.swift

```diff
@@ -0,0 +1,71 @@
+import Foundation
+import UIKit
+
+class DayForecastWeatherCell: UICollectionViewCell {
+    private var dayLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = .custom(.white)
+        label.font.withSize(10)
+        return label
+    }()
+
+    private var temperatureLabel: UILabel = {
+        let label = UILabel()
+        label.text = "12 C"
```

Comment:
I suppose to, default value should have `°C`

Reply:
👍